generator/insn_vdvs2vmp3.go

```diff
@@ -30,22 +30,26 @@ func (i *Insn) genCodeVdVs2VmP3(pos int) []string {
 		builder.WriteString(i.gLoadDataIntoRegisterGroup(0, c.LMUL1, c.SEW))
 
 		vs2EEW := c.SEW / SEW(f)
-		if vs2EEW > SEW(i.Option.XLEN) {
+		if vs2EEW < 8 {
 			res = append(res, "")
 			continue
 		}
 
-		vdEMUL := c.LMUL * LMUL(f)
+		vdEMUL := c.LMUL / LMUL(f)
```

Comment:
`vdEMUL` does not make sense anymore, `vs2EMUL := c.LMUL / LMUL(f)` should be used.